packages/in_app_purchase/lib/src/store_kit_wrappers/sk_product_wrapper.dart

```diff
@@ -20,28 +19,29 @@ part 'sk_product_wrapper.g.dart';
 class SkProductResponseWrapper {
   /// Creates an [SkProductResponseWrapper] with the given product details.
   SkProductResponseWrapper(
-      {@required this.products, @required this.invalidProductIdentifiers});
+      {required this.products, required this.invalidProductIdentifiers});
 
   /// Constructing an instance from a map from the Objective-C layer.
   ///
   /// This method should only be used with `map` values returned by [SKRequestMaker.startProductRequest].
   /// The `map` parameter must not be null.
```

Comment:
Remove this comment; it's now redundant.

Reply:
done, and removed all the similar documentations.